### PR TITLE
Fast redirection after successfully saving an article. Don't re-enabl…

### PR DESCRIPTION
**Pull request by paulocheque**
…e the buttons in case of success. Also, renamed the Cancel button to Back.

### CODE REVIEW

File: pskb_website/static/js/editor_utils.js [hunk 1]
```diff
@@ -437,21 +437,25 @@ function save(sha, path, secondary_repo) {
         dataType: 'json',
         cache: false,
         beforeSend: function(xhr) {
-            $('.btn-save').prop('disabled', true);
             $('html, body').css("cursor", "wait");
+            $('#editor-options').prop('disabled', true);
```
Comment (durden) [inline pskb_website/static/js/editor_utils.js:464]: Good catch!  It's good to disable the options during send as well!

**Comment by durden** [comment]
Any others ideas here or should I go ahead and merge?

**Comment by paulocheque** [comment]
I was trying to think about disabling the button when the title was empty. But I did not find a elegant solution yet, because we need to consider other variables.

**Comment by durden** [comment]
Ah ok, not a problem.  I just wanted to make sure it shouldn't be merged yet.

**Comment by paulocheque** [comment]
Added a enable/disable function of the save button. It is fine to merge for me.

Test on `guides-dev`. Rebased with the `origin/master`. I can rebase with the `heroku/master` too.

**Comment by durden** [comment]
I tested this a bit, then merged it.  Then, I found some issues so I created another PR to revert it.  So, the history, etc. is sort of messed up.  Guess there's no way to 're-open' a PR on github.  Either way, let me describe the issues and you can create another PR that fixes it.  Sorry for the confusion!

All the changes look good except the save button tooltip.  Maybe these issues have something to do with using `destroy` on the tooltip?

Here are the weird issues I've seen with the tooltip:
1. I cannot always get it to show up when I hover over the save button.
   - Try this to reproduce:
     - Immediately reload the page and hover over the save. This works.
     - Now reload the page and wait a few seconds **before** hovering over the save.  I cannot get the tooltip to show up until I hover once, hover away, then hover back.
2. Sometimes the tooltip on the save button gets stuck. It's hard to see in this screenshot, but my cursor is in the editor window but the save button tooltip is still visible.
   screenshot 2016-05-05 10 04 37
3. Another example of the tooltip being stuck. I'm hovering over the options button but the save button tooltip is still visible:
   screenshot 2016-05-05 10 11 38
4. The code references a different tooltip on the save button when the live markdown preview is visible, but I cannot ever seem to see this tooltip at all when the markdown preview is displayed.

**Comment by paulocheque** [comment]
Ok I will open a new PR for this.
I believe it is related to the disabled button. Tooltips does not work well with disabled buttons. But I will check the destroy part too.

**Pull request merged**